docs/configuration/config-file-reference.md

```diff
@@ -4371,6 +4375,18 @@ ring:
 # Disable the rule_group label on exported metrics
 # CLI flag: -ruler.disable-rule-group-label
 [disable_rule_group_label: <boolean> | default = false]
+
+# Enable high availability
+# CLI flag: -ruler.enable-ha
+[enable_ha: <boolean> | default = false]
```

Comment:
`enable_ha` feels like a bad name to me. Can we just replace this with checking replication factor?
If we can't, can we be specific about what HA we want to enable? For example, `enable_ha_evaluation`

Reply:
API HA checks only for replication factor. My thought was to use a separate flag in case if someone wants to use one or the other and not both. I can change this to `enable_ha_evaluation`